server/routes/index.js

```diff
@@ -1,12 +1,21 @@
 const express = require('express');
 const path = require('path');
+const getVotersFromDB = require('../helpers/getVotersHelper');
 
 const router = express.Router();
-const voters = require('../model/voters');
+// const voters = require('../model/voters');
```

Comment:
why is this commented out ?

Reply:
we are using this for tests( the connection between the back end and the front end), replacing the database.